### ISSUE TITLE
Onderzoeken: De layerdata wordt tweemaal opgehaald voor een enkele page view

### ISSUE THREAD

**Issue by BaDu-HSN** [Hoogspanningsnet/Webkaart-scripts#66]
Dit lijkt wat dubbel op en genereert een boel overbodige data.
Ligt dit aan de hash generator?

of aan de accuracy aantal decimalen van de URL query? (die kunnen trouwens wel wat minder)
Zie regels 74 en 75 van GeoJSON.Ajax.js

**Comment by BaDu-HSN** [comment]
Lijkt alleen voor te komen bij een volledige reload van de kaart URL.
Bij zoomen pannen is er slechts een load.
